docker-compose.tendermint.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+
```

Comment:
why do we need a sparate file for single node? we can just have one compose file and different sets of commands in readme file to bring up one node or multi node

Reply:
Because some files such as `genesis.json` and `config.toml` differ for each case: standalone versus cluster.